### PR TITLE
allow tolerations to be passed to fluentbit containers

### PR DESCRIPTION
**Pull request by paynejacob**
PROBLEM:

fluentbit pods are not installed on controlplane and etcd nodes

SOLUTION:

allow tolerations to be passed to all fluentbit pods created by rancher.

### CODE REVIEW

**Review by rmweir — APPROVED** [review]
LGTM

File: packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ spec:
       Tag: "eks"
       Path: "/var/log/messages"
       Parser: "syslog"
+    {{ if .Values.fluentbit_tolerations }}
```
Comment (mrajashree) [inline packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml:19]: I think this `if` is redundant? same for the other places

Comment (paynejacob) [reply packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml]: good catch.

**Review by mrajashree — APPROVED** [review]

**Review by cbron — CHANGES_REQUESTED** [review]
remove tgz's

**Review by cbron — APPROVED** [review]

**Pull request merged**